modules/tensor_mechanics/test/tests/domain_integral_thermal/interaction_integral_2d_bf.i

```diff
@@ -0,0 +1,126 @@
+#This is a regression test that exercises the option to include
+#the effects of the body force in the interaction integral. This
+#uses the same basic model as the other cases in this directory.
+#This is a placeholder until a suitable problem with an analytical
+#solution can be identified.
+
+[GlobalParams]
+  order = FIRST
+  family = LAGRANGE
```

Comment:
this + `add_variables = true` is maybe a bit of a dicey mix, since the TMA determines the appropriate variable order automatically. FIRST/LAGRANGE is the default for NL and Aux variables anyways, so what block is this applying to, the `DomainIntegralAction` (which is using `std::string` as parameter types for those 👎 )?
As a maintenance task we might want to fix that type of stuff sometime.

Reply:
I'll just remove order/family from all of the cases affected by this PR.